dts/arm/st/u0/stm32u0.dtsi

```diff
@@ -389,7 +396,8 @@
 		rng: rng@40025000 {
 			compatible = "st,stm32-rng";
 			reg = <0x40025000 0x400>;
-			clocks = <&rcc STM32_CLOCK(AHB1, 18)>;
+			clocks = <&rcc STM32_CLOCK(AHB1, 18)>,
+				<&rcc STM32_SRC_CK48 NO_SEL>;
```

Comment:
Nitpicking on indentation (there were all fixed not so long ago :-)
```c
			clocks = <&rcc STM32_CLOCK(AHB1, 18)>,
				 <&rcc STM32_SRC_CK48 NO_SEL>;
```
Ditto below.